firebase/functions/src/comments.ts

```diff
@@ -85,4 +108,20 @@ function addToFIreBase(data: any, collection: any) {
       });
   });
 }
-//943.05
+
+function update(data: any, collection: any,id:any) {
+  return new Promise((resolve, reject) => {
+      const userRef = db.collection(collection).doc(id)
+      userRef.update(data).then(function () {
+          resolve({
+              res: '200'
+          })
+      }).catch(function (error) {
+          // The document probably doesn't exist.
+          console.error("Error updating document: ", error);
```

Comment:
More detailed message should be considered.
```suggestion
          console.error(`Error while updating document /${collection}/${id} with error:`, error);
```